### ISSUE TITLE
[Tiled Gallery] Block Styles Images Change Size 

### ISSUE THREAD

**Issue by bobmatyas** [Automattic/jetpack#22900]
### Impacted plugin

Jetpack

### Steps to Reproduce

1. Create a test site.
2. Install Jetpack
3. Add a new post
4. Add a "Tiled Gallery"
5. Observe that the images in "Block Styles" constantly change size

### A clear and concise description of what you expected to happen.

Images should remain a static size in block style options

### What actually happened

Images rapidly increase and decrease in size in block style options.

See video:

https://user-images.githubusercontent.com/45246438/154124844-f6c2d9b0-867c-4846-867c-e1e4116a9677.mp4

### Operating System

macOS

### Browser

Firefox

### Browser Version(s)

v98

**Comment by jeherve** [comment]
Thanks for the report. do you have any other plugins, such as Gutenberg, installed on the site?

**Comment by bobmatyas** [comment]
No, only Jetpack on a fresh install of WP 5.9 w/default theme (JN site).

**Comment by sophiegyo** [comment]
I was able to replicate this on a fresh self-hosted test site (WP 5.9.2, Jetpack 10.7, nothing else) now. Couldn't replicate on Atomic or Simple wpcom. I was actually testing for something else but figured it was worth dropping here since I happened on this bug report while looking for any existing report for my actual issue.

Browser: Firefox 98.0
OS: 11.6 Big Sur

https://user-images.githubusercontent.com/36608681/160764386-32ba9274-5722-4f53-8b44-6628d104d06a.mp4

**Comment by jeherve** [comment]
Closing as duplicate of #11783
